packages/types/src/wallet.ts

```diff
@@ -23,22 +28,27 @@ export type Custody = HotWallet; // Later on, could have different types (like l
 export class Wallet {
   constructor(
     readonly label: string,
-    readonly id: string,
-    readonly fullViewingKey: string,
+    readonly id: WalletId,
+    readonly fullViewingKey: FullViewingKey,
```

Comment:
Wallet is stored in zustand memory, we can use here bufbuild types `FullViewingKey` and `WalletId`